src/Uno.UI.XamlHost.Skia.Wpf/Uno.UI.XamlHost.Skia.Wpf.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
 
 	<PropertyGroup>
-		<TargetFrameworks>net7.0-windows;net8.0-windows</TargetFrameworks>
+		<TargetFrameworks>$(NetPrevious)-windows;$(NetCurrent)-windows</TargetFrameworks>
```

Comment:
Why no `NetWindowsPreviousAndCurrent` - like there is with Skia & WASM?

Reply:
I should add this property, but I think having `Wpf` in the name would be clearer to not confuse it with UWP/WinUI TFMs (uap and netX.0-windows10.0.xxxx)